runtests.py

```diff
@@ -205,8 +205,8 @@ def add_myunit(driver: Driver) -> None:
         mod = file_to_module(f)
         if '.codec.test.' in mod:
             # myunit is Python3 only.
-            driver.add_python_mod('unittest %s' % mod, 'unittest', mod)
-            driver.add_python2('unittest %s' % mod, '-m', 'unittest', mod)
+            driver.add_python_mod('unittest %s' % mod, 'unittest', '-q', mod)
```

Comment:
It'd like to see this in a separate pull request, or a mention in the description of the pull request, as this is logically independent from the rest of this PR. No need to anything about this right now, but maybe next time.